docs/how-to/h-kraft-mode.md

```diff
@@ -0,0 +1,53 @@
+# How to set up KRaft mode
+
+> **IMPORTANT** This feature is experimental and not production-ready. UX can change significantly in the upcoming months.
```

Comment:
I think we should abstain from using **NOTES**, **WARNING**, or **IMPORTANT** if possible after we convert the notes to a proper syntax. The caution type has an exclamation mark in a triangle Icon. But if you really want to use it ... not a biggie. In this particular case, just a caution-type note seems perfect.